pixeltable/catalog/catalog.py

```diff
@@ -1214,12 +1214,17 @@ def __store_replica_md(self, path: Path, md: schema.FullTableMd) -> None:
                 ),
             )
             conn.execute(q)
-        else:
-            assert existing_md_row.md['is_replica']
-            if md.tbl_md.current_version > existing_md_row.md['current_version']:
-                # New metadata is more recent than the metadata currently stored in the DB; we'll update the record
-                # in place in the DB.
-                new_tbl_md = dataclasses.replace(md.tbl_md, name=path.name, user=Env.get().user, is_replica=True)
+        elif not existing_md_row.md['is_replica']:
+            raise excs.Error(
+                'An attempt was made to replicate a view whose base table already exists in the local catalog '
```

Comment:
Is this making assumptions about the state of the caller? Maybe this should be checked in the caller?

Also, does that mean that the following won't work?
- I publish T
- someone else replicates T and creates a view V
- I replicate V